pkg/insightstore/report.go

```diff
@@ -0,0 +1,305 @@
+// Copyright 2020 The PipeCD Authors.
```

Comment:
nit: Update the file name, it should be `chunk.go` and `chunk_test.go`.

Reply:
Oops 🙊